pkgs/development/tools/coursier/default.nix

```diff
@@ -1,18 +1,25 @@
-{ stdenv, fetchurl, makeWrapper, jre }:
+{ stdenv, fetchurl, makeWrapper, jre, writeScript, common-updater-scripts
+, coreutils, git, gnused, nix, nixfmt }:
 
 let
-  zshCompletion = version: fetchurl {
-    url = "https://raw.githubusercontent.com/coursier/coursier/v${version}/modules/cli/src/main/resources/completions/zsh";
+  version = "2.0.6";
+
+  zshCompletion = fetchurl {
+    url =
+      "https://raw.githubusercontent.com/coursier/coursier/v${version}/modules/cli/src/main/resources/completions/zsh";
     sha256 = "0afxzrk9w1qinfsz55jjrxydw0fcv6p722g1q955dl7f6xbab1jh";
   };
-in
-stdenv.mkDerivation rec {
+
+  repo = "git@github.com:coursier/coursier.git";
+in stdenv.mkDerivation rec {
+  inherit version;
+
   pname = "coursier";
-  version = "2.0.6";
 
   src = fetchurl {
-    url = "https://github.com/coursier/coursier/releases/download/v${version}/coursier";
-    sha256 = "0zg07bi44a5p0zfc77i1vmjz9rwzxzf6p0xsybsv3n1saq884wij";
+    url =
+      "https://github.com/coursier/coursier/releases/download/v${version}/coursier";
```

Comment:
```suggestion
    url = "https://github.com/coursier/coursier/releases/download/v${version}/coursier";
```

Reply:
@SuperSandro2000 What is this running? `nixfmt` disagrees with these suggestions.

Reply:
That this line break does not really add anything in keeping the line shorter. Also a line length of 80 is really outdated and only uses 1/5 of most monitors.